### PR TITLE
Disabled polygon editing for SYNCED plant locations

### PR DESCRIPTION
**Pull request by ankitecd**
Changes in this pull request:
- disabled editing of polygon for SYNCED plant locations for release
- added HID on review screen for SYNCED plant locations
- added hid, locationId, sampleTreesCount and uploadedSampleTreesCount in GeoJSON export
- fixed Free up space not appearing on uploaded registrations page

### CODE REVIEW

**Comment by tejassonar** [comment]
Not able to see hid on review screen for both single and multiple trees, even when registrations are uploaded.
image
image
image

getting hid in the GeoJSON export though.

**Comment by tejassonar** [comment]
Also, all the plant locations are not getting retrieved. Not sure if this issue is related to this PR, but this should be fixed.

File: app/actions/uploadInventory.ts [hunk 1]
```diff
@@ -538,12 +544,13 @@ const checkSampleTreesAndUpload = async (inventory: any) => {
         } else if (response && response.coordinates[0].status === 'complete') {
           sampleTree.status = SYNCED;
           sampleTree.locationId = response.id;
+          sampleTree.hid = response.hid;
```
Comment (norbertschuler) [inline app/actions/uploadInventory.ts:547]: - [ ] I am not getting an HID codes displayed on iOS simulator while testing this PR - not for sample trees and not for the registration itself:

Simulator Screen Shot - iPhone 13 - 2022-01-15 at 17 10 58

**Review by norbertschuler — COMMENTED** [review]
- [ ] The edit button is available on the review screen while registering before you save the registration. But you can not edit the registration afterwards anymore - even if it is not yet uploaded (synced) to the server. Is this the wanted behavior?

Simulator Screen Shot - iPhone 13 - 2022-01-15 at 17 15 21Simulator Screen Shot - iPhone 13 - 2022-01-15 at 17 16 12

**Comment by ankitecd** [comment]
> * [ ]  The edit button is available on the review screen while registering before you save the registration. But you can not edit the registration afterwards anymore - even if it is not yet uploaded (synced) to the server. Is this the wanted behaviour?
> 

Yes the upload process is not context aware. So it takes all the registrations and starting uploading, and ignores any later changes. So if something is in editing state the already running upload process doesn't know that. We have to change this implementation to make it context aware.

**Comment by norbertschuler** [comment]
@ankitecd do you want to merge develop into this PR and look at the issues above?

**Review by tejassonar — APPROVED** [review]

**Comment by tejassonar** [comment]
LGFM

File: app/actions/uploadInventory.ts (hunk 1 above)
Comment (norbertschuler) [reply app/actions/uploadInventory.ts:567]: At least after reloading the data from the server all HID's are displayed.

**Review by norbertschuler — APPROVED** [review]
The described fixes work for me.

**Pull request merged**